packages/std-contracts/src/libraries/LibStorage.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { IUint256Component } from "solecs/interfaces/IUint256Component.sol";
+import { IWorld } from "solecs/interfaces/IWorld.sol";
+import { getSystemAddressById, getAddressById } from "solecs/utils.sol";
+
+uint256 constant worldID = uint256(keccak256("mud.world"));
+uint256 constant componentsID = uint256(keccak256("mud.components"));
+
+/**
+ * This library will only work if you use `std-contracts/StdSystem.sol` for every system in your contracts
+ */
+
+library LibStorage {
```

Comment:
Minor naming nit - wdyt about calling this `SystemStorage`? This is a pattern used by `@solidstate` (eg https://github.com/solidstate-network/solidstate-solidity/blob/master/contracts/access/ownable/OwnableStorage.sol) and makes it clearer what storage this library manages. (I can see us adding more different types of storage in the future and probably want to manage them in individual libraries)

Reply:
Or `LibSystemStorage` if you prefer. Also why did you remove the `system(uint256 id): address` and `component(uint256 id): address` utils from the library? They seem useful to reduce boilerplate.

Reply:
Renamed to `SystemStorage` and added `system` and `component` getters in [16e2033](https://github.com/latticexyz/mud/pull/264/commits/16e203374934ad3df1115076c0083939d69a1060)